tests/ext/sandbox-prehook/close_on_exit.phpt

```diff
@@ -0,0 +1,45 @@
+--TEST--
+[Prehook Regression] Run sandbox closures for open spans on exit
+--SKIPIF--
+<?php if (PHP_VERSION_ID < 70000) die('skip: Prehook not supported on PHP 5'); ?>
+--FILE--
+<?php
+use DDTrace\SpanData;
+
+register_shutdown_function(function () {
+    $spans = dd_trace_serialize_closed_spans();
+    array_map(
+        function($span) {
+            echo @$span['name'], PHP_EOL;
```

Comment:
What's the reason for suppressing this?
```suggestion
            echo $span['name'], PHP_EOL;
```

Reply:
Not sure of the original reason, but this is copied over as-is from the original test: https://github.com/DataDog/dd-trace-php/blob/master/tests/ext/sandbox/close-on-exit.phpt#L13 I like your suggestion though. :)